nservicebus/upgrades/5to6.md

```diff
@@ -234,6 +234,7 @@ The `IMessageMutator` and `IMutateTransportMessages` interfaces are no longer av
 
  * `IncomingContext` and `OutgoingContext` have been replaced with multiple contexts ("stages") per pipeline offering more useful information depending on the requirements. [Pipeline Steps Stages and Connectors](/nservicebus/pipeline/steps-stages-connectors.md) provides more details about the available stages.
  * `Behavior<TContext>` now provides the base class for implementing custom behaviors and replaces `IBehavior<TContext>`.
+ * All behaviors are effectively singletons, no matter how they are registered in the container.
```

Comment:
"no matter how they are registered in the container"  what is the value of this? do we want to say "behaviors are not resolved from the container and hence should not be registered in the container"?